Kitodo-DataFormat/src/main/java/org/kitodo/dataformat/access/MetsXmlElementAccess.java

```diff
@@ -399,4 +420,123 @@ private StructLink createStructLink(LinkedList<Pair<String, String>> smLinkData)
         }).collect(Collectors.toList()));
         return structLink;
     }
+
+    private void addChildLinks(Map<String, LinkedMetsResource> childLinks, Mets mets) {
+        Map<String, Collection<Entry<String, LinkedMetsResource>>> linksByParent = collectTheLinksPerParent(childLinks);
```

Comment:
I think that is always a bit superfluous for a local variable, which specifies the results of a private method, which is used in just one other method, to specify its own object ... but as you like.

Reply:
I find it more easy to read in the future. At this moment it is used only in this place, but can you be 100% sure that it will be not extended in the future?

Thank you for change.